### ISSUE TITLE
Apply styling

### ISSUE THREAD

**Issue by sclavijo93** [soldair/node-qrcode#239]
Hi, I've searched for this question but I didn't find anything among the issues.

I'm wondering if there is any way to apply styles to the QR? Thanks in advance

**Comment by primecodecom** [comment]
Using [c++](https://github.com/nayuki/QR-Code-generator/tree/master/cpp) implementation of library, I made some changes. 
[Updated library is here
](https://www.npmjs.com/package/@primecode/async-qrcode)
1. Update the code to work with AsynWorker, so the code could execute in different thread and build it to node js addon. 
2. Added 3 different type of style dots 
 <img src="https://github.com/primecodecom/files/raw/master/ROUND.png" width="120" height="120">
<img src="https://github.com/primecodecom/files/raw/master/SQUARE.png" width="120" height="120">
<img src="https://github.com/primecodecom/files/raw/master/CIRCLE.png" width="120" height="120">
3. Different color styles
<img src="https://github.com/primecodecom/files/raw/master/COLORED.png" width="120" height="120">

**Comment by DYW972** [comment]
Thank your for your work @primecodecom  ! 👏

This is one is particularly lit !

> <img alt="" width="120" height="120" src="https://github.com/primecodecom/files/raw/master/ROUND.png">
> 


But can you be more specific for this comment _"// The QR Code can tolerate about 15% erroneous codewords"_ in your export types variable at` index.d.ts` ?

```
export const Types: {
CIRCLE: 0,  // The QR Code can tolerate about  7% erroneous codewords
ROUND: 1,  // The QR Code can tolerate about 15% erroneous codewords
SQUARE: 2
};
````
What about performance and security ?

Any suggestions appreciate !

Best regards

**Comment by bradisbell** [comment]
Has anyone seen a pure JavaScript implementation?  Using this library client-side... would be great to see more stylized SVG options.

**Comment by BenRacicot** [comment]
Yes! This! But further, I'm seeing some implementations simply place a logo over the center of the QR image. 
It would seem that devs are cranking up the error correction tolerance to keep the code working?

Perhaps this would be a different issue but **allowing an image in the center and the code to be actually built around it would be an incredible style upgrade**.
